jetty-core/jetty-server/src/main/config/modules/static-deploy.mod

```diff
@@ -0,0 +1,20 @@
+[description]
+# tag::description[]
+Scans and deploys `static` contexts from `$JETTY_BASE/webapps/` directory.
```

Comment:
What's the difference between `static-deploy` and `core-deploy`???

Reply:
* `core-deploy` is for robust deployments with a Handler, optional `libs/`, optional `classes/`, optional `static/` directories.  A Classloader, etc.
* `static-deploy` is for static content deployment.

Reply:
That is going to need both javadoc and documentation as that is a fine distinction to make. Why isn't the `static-deploy` a degenerate case of `core-deploy`???

Reply:
The structure is different.
The original implementation in this PR did exactly that, by making static `/foo/` work, and robust work with `/foo.d/`, but greg wanted only 1 directory type, not the ".d" version.
Detecting the difference with only 1 name was very tricky (and prone a security issue with accidental revealing of non-static content in a static way), and it was decided to just split the concept up between `static` environment and `core` environment instead.

Reply:
Documentation is a must, and is being done by simone in a different branch.
This specific scenario, supporting static, from non-ee#, was a part in the documentation that needed addressing, hence the recent issue and this PR.

Reply:
Can we work on the javadoc/descriptions to make the distinction really clear:
  + static - Jetty will provide the context/handlers to serve static content and only static content
  + core - The application must provide and/or configure handlers to serve static and/or dynamic content.